### ISSUE TITLE
wrong path "Branches Aren't Just For Birds"

### ISSUE THREAD

**Issue by AndreaBLS** [jlord/patchwork#33516]
Section - Branches Aren't Just For Birds

"Save this file in the 'contributors' folder in Patchwork: Patchwork/contributors/add-yourusername.txt"

when verifying , i get "File not in contributors folder"  , even though i am in the right folder.
Repo got messed from random merges into master, so many wrong directories!

**Comment by avstoyanov** [comment]
to me, it seems that the directory somehow got capitalized.

**Comment by murpholinox** [comment]
Yes, I think that is the issue. I try to fix this but I do not have proper permissions.... I think (I am a beginner with this git-stuff) @jlord. Same as #30434 and #27932

**Comment by derekb123** [comment]
I am having the same issue, did anyone get this resolved? it looks like there is no "contributors" directory just "Contributors" or "CONTRIBUTORS" @jlord

**Comment by lisafaggiolo** [comment]
Still hasn't been fixed which makes one of the "git-it verify" checks fail. What I did to solve it after trying different things out, was to simply(lol) create my own "contributors" folder spelled exactly like the "git-it verify" wanted it spelled. AND IT WORKED! So yeah it took a minute there but hopefully this can help others!

**Comment by serahmetpalli** [comment]
I am finding this super strange, I had the same issue as well with the two contributors folders...so I created my own folder...same error :/

**Comment by lisafaggiolo** [comment]
So what I did is I deleted the whole "patchwork" fork  along with my add-lisafaggiolo first and started over from the "forks and clones" steps in git-it. I then created the contributors folder at the same time as my add-lisafaggiolo inside the new "patchwork" fork and I was able to add my file contributors/add-lisafaggiolo, push my branch and complete the task. In later steps reporobot told me that I had too many files, I did what was prompted and delete it no problem since I did push my branch anyway. Hopefully this will help!

**Comment by tundlestein** [comment]
I had the same issue, all I did was to open the Git-Hub desktop app and used that to push instead. after doing that I verified the step within Git-it and it cleared fine.
I have no idea what's causing the issue but using the app instead of the cmd line to force it through worked for me.

**Comment by siammais10** [comment]
I had this issue. I resolved by placing the file in the CONTRIBUTORS folder and the CONTRIBUTORS/contributors folder .... hopefully bug gets fixed soon

**Comment by ViktorB979** [comment]
Worked for me as well  by copying the file in CONTRIBUTORS/contributors folder.
